app/views/admin/comments/index.html.erb

```diff
@@ -1,59 +1,57 @@
-<main>
```

Comment:
Perhaps @jacobherrington can comment on this, but I remember reviewing [this PR](https://github.com/forem/forem/pull/9927/files) where <main> was added to fix general accessibility warnings

Reply:
The `<main />` tag is used to identify the primary content of the page. It can be the `<main />` tag or `role='main'`. See the [Main Landmark](https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/main.html) documentation for more info. Having this tag or role helps assistive technologies navigate a page.

Reply:
Thanks Nick 😊 `Sounds like we can add the `role='main'` back at least so as to not reverse the work done in the PR to fix the general accessibility warnings in admin. I don't remember seeing these warnings, when/where/how do we see them?

Reply:
Hmm, I do have this tag but somewhere else now - in [general layout](https://github.com/forem/forem/pull/10015/files#diff-c861d9ec1cc861e6b11cab02955eaaeaR42) - do you think that's wrong?

Reply:
Awesome, I'm not sure exactly how this works without doing more research, so maybe @jacobherrington or @nickytonline can verify that the changes make sense.